config/webpack-config.js

```diff
@@ -10,6 +10,11 @@ const TerserPlugin = require( 'terser-webpack-plugin' );
 // Constants
 const COMMON_BUNDLE_NAME = 'common.js';
 
+/* This sets the default mode for webpack configurations to satisfy the need
+   of webpack to have a `mode` set.
+   This value gets overridden when NODE_ENV=development. */
```

Comment:
```suggestion
   This value gets overridden when NODE_ENV=development.
   See the `if ( envvars.NODE_ENV === 'development' )` block below. */
```

Reply:
Not super-ly a fan of the literal code in the comment since it becomes two things that need to be kept in sync. But if it helps 👍